src/components/auth0-login-zone/auth0-login-zone.js

```diff
@@ -23,7 +23,7 @@ export const Auth0LoginZone = ({ auth: { isAuthenticated, isLoading, loginWithRe
   const mouseEnterMenu = () => { if (!isMobile) { setMouseInMenu(true) } }
   const mouseExitMenu = () => { if (!isMobile) { setMouseInMenu(false) } }
 
-  const toggleMenu = () => { if (isMobile) { setMenuToggled(!isMenuToggled) } }
+  const toggleMenu = () => { setMenuToggled(!isMenuToggled) }
```

Comment:
Oh this is interesting. I hadn't even considered that we might care about mobile devices. Cool that this was an easy fix, just allowing the toggle on all devices